src/entity/user/User.ts

```diff
@@ -66,7 +67,6 @@ export class User {
   name: string;
 
   @Column({ type: 'text', nullable: true })
-  @Index('IDX_user_email')
```

Comment:
Don't we want to keep one of the regular email indexes, probably this one?

Reply:
Re-added